Release 0.8.2 (#78)
matter-ecs/matter: CHANGELOG.md

## [Unreleased]

## [0.8.2] - 2024-06-25

### Fixed

> - Fixed the `:view()` arm on an empty Query to return a function rather than a handle.

Reviewer: ```suggestion
- Fixed `view` on an empty query being a table instead of a function
```

> - Fixed the `:view()` arm on an empty Query to return a function rather than a handle.

Reviewer: ```suggestion
- Calling `:view()` on an empty query will no longer error.
```

> - Snapshot now correctly returns an empty list rather than nil on an empty Query.

Reviewer: ```suggestion
- Fixed `:snapshot()` on an empty query returning nil instead of an empty list
```

> - Snapshot now correctly returns an empty list rather tahn nil on an empty Query.l

Reviewer 2: typo
